src/pygeosx/pyhistory/PyHistory.cpp

```diff
@@ -0,0 +1,60 @@
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+
+// Source includes
+#include "../pygeosx.hpp"
+#include "PyHistory.hpp"
+
+#include "PyHistoryCollectionType.hpp"
+#include "PyHistoryOutputType.hpp"
+
+namespace geosx
+{
+extern std::unique_ptr< GeosxState > g_state;
```

Comment:
Hmm, I'm really not a fan of `extern` variables. Could you not just call `getGlobalState` in this file instead of using `g_state` directly?

Also if we'll need to check if things are already initialized we should make a function in `pygeosx.hpp` instead of directly accessing the variable.

Reply:
Yes I forgot to remove this in this file. Thanks for catching those mistakes...